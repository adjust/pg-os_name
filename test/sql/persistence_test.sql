BEGIN;
-- persistence should persist os_names;
-- ./spec/persistence_spec.rb:8;
CREATE EXTENSION os_name;
CREATE TABLE os_name_test AS SELECT 'ios'::os_name, 1 as num;
SELECT * FROM os_name_test;
UPDATE os_name_test SET num = 2;
SELECT * FROM os_name_test;
ROLLBACK;
