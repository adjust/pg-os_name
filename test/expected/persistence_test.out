BEGIN;
-- persistence should persist os_names;
-- ./spec/persistence_spec.rb:8;
CREATE EXTENSION os_name;
CREATE TABLE os_name_test AS SELECT 'ios'::os_name, 1 as num;
SELECT * FROM os_name_test;
 os_name | num 
---------+-----
 ios     |   1
(1 row)

UPDATE os_name_test SET num = 2;
SELECT * FROM os_name_test;
 os_name | num 
---------+-----
 ios     |   2
(1 row)

ROLLBACK;
